### PR TITLE
Basic interop support for Ecs::Graphics

### PR DESCRIPTION
**Pull request by jeuxjeux20**
This PR simply serves as a sort of baseline to initiate the development of the Interop API for `NovelRT::Ecs::Graphics`. This should help understanding how the C API works and adding features to it!

### Added things
- The `DefaultRenderingSystem` handle type, with `FindInScheduler` and `ForceVertexTextureFutureResolution` methods

### CODE REVIEW

File: include/NovelRT/NovelRT.h [hunk 1]
```diff
@@ -155,12 +155,6 @@
   #include <NovelRT.Interop/Ecs/NrtUnsafeComponentView.h>
   #include <NovelRT.Interop/Ecs/Audio/NrtEcsAudio.h>
 
-#ifdef NOVELRT_INK
-  // Ink
-  #include <NovelRT.Interop/Ink/NrtInkService.h>
-  #include <NovelRT.Interop/Ink/NrtStory.h>
-#endif
-
```
Comment (RubyNova) [inline include/NovelRT/NovelRT.h:163]: Why is this in the PR?

File: include/NovelRT/NovelRT.h [hunk 2]
```diff
@@ -184,9 +178,6 @@
 
   // Utilities
   #include <NovelRT.Interop/Utilities/NrtMisc.h>
-
-  // Windowing
-  #include <NovelRT.Interop/Windowing/NrtWindowingService.h>
```
Comment (RubyNova) [inline include/NovelRT/NovelRT.h:189]: Why is this in the PR?

**Review by RubyNova — CHANGES_REQUESTED** [review]

File: include/NovelRT/NovelRT.h (hunk 2 above)
Comment (jeuxjeux20) [reply include/NovelRT/NovelRT.h:189]: Related to #484, else using the `NovelRT.h` header with Interop would just not compile which is sad.

Comment (RubyNova) [reply include/NovelRT/NovelRT.h:189]: Its already merged in main though?

**Review by RubyNova — CHANGES_REQUESTED** [review]

File: include/NovelRT.Interop/NrtTypedefs.h [hunk 3]
```diff
@@ -71,6 +71,7 @@ extern "C"
 #include "DotNet/NrtDotNetTypedefs.h"
 #endif
 #include "Ecs/Audio/NrtEcsAudioTypedefs.h"
+#include "Ecs/Graphics/NrtEcsGraphicsTypedefs.h"
 #include "Ecs/NrtEcsTypedefs.h"
 #ifdef NOVELRT_INK
 #include "Ink/NrtInkTypedefs.h"
```
Comment (RubyNova) [inline include/NovelRT.Interop/NrtTypedefs.h:77]: GH won't let me suggest the change but can we delete the Ink stuff here?

Comment (jeuxjeux20) [reply include/NovelRT.Interop/NrtTypedefs.h]: Done

**Review by RubyNova — APPROVED** [review]
LGTM

**Pull request merged**